src/initial-submission/components/FileDetailsForm.tsx

```diff
@@ -42,14 +46,38 @@ const FileDetailsForm = ({ initialValues, schemaFactory, ButtonComponent }: Step
     });
     const schema = schemaFactory(t);
 
+    const validationResolver = useCallback((data: FileDetails) => {
+        try {
+            schema.validateSync(
+                { ...data, manuscriptFile: manuscriptStatus.fileStored.fileName },
```

Comment:
Does the current schema check the manuscriptFile property already?

Reply:
I added it to the existing schema

Reply:
I must have missed that on my firstpass. Looks good